src/api/app/models/binary_release.rb

```diff
@@ -235,12 +235,12 @@ def update_for_product
 #  binary_epoch              :string(64)       indexed => [binary_name, binary_version, binary_release, binary_arch]
```

Comment:
It seems `binary_epoch` also could have a default, but I'm still not sure...

Reply:
It's not set in the code so far so I wouldn't do this...